airbyte-integrations/connectors/source-mongodb-v2/build.gradle

```diff
@@ -38,8 +38,8 @@ java {
 }
 
 dependencies {
-    implementation 'io.debezium:debezium-embedded:2.5.1.Final'
-    implementation 'io.debezium:debezium-connector-mongodb:2.5.1.Final'
+    implementation 'io.debezium:debezium-embedded:2.6.0.Final'
```

Comment:
Perhaps using the current stable 2.6.2.Final?
I see a few [fixes](https://debezium.io/releases/2.6/release-notes) for mongodb